src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```diff
@@ -136,6 +136,10 @@ const ImageClassificationObservationTable = ({ uploadedFiles, handleRemoveFile }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [projectId, recordId])
 
+  const _updateFilesOnUpload = useEffect(() => {
```

Comment:
update table as images are available

Reply:
ah yes see the issue here - will update to separate processed images from new images